source/license.html

```diff
@@ -11,16 +11,31 @@
 	<article>
 	  <header><h2>License</h2></header>
 	  <div class="entry-content">
-	    <p>
-	    This is a copy of the current LICENSE file from
-	    the main repository.
-	    The plain-text document version of this document is available
-	    here:
-	    <a href="license.txt">license.txt</a>
-	    </p>
-	    <pre>
-	    <!--#include virtual="license.txt" -->
-	    </pre>
+            <p>
+            OpenSSL is covered by one of two licenses, depending on which
+            release is involved.  In all cases, there is a file named
+            <code>LICENSE</code> in the top-level of the release.
+            Copies can also be found here.
+            </p>
+            <p>
+            For the 1.1.1 release, and later releases derived from that,
+            <a href="license-asf.txt">the Apache v2 license</a>
```

Comment:
Would you mind having the version number in the file name?  Also, I've heard repeatedly from Apache folks that "a*s*f" is an incorrect acronym, so it might be better not to use an acronym at all.  `apache-license-2.0.txt`, perhaps?